Symbols.csv

```diff
@@ -1664,7 +1664,7 @@ F1DE,,sliders
 F1E0,,share
 F1E1,,share square solid
 F1E2,,bomb
-F1E3,,futbol soccer ball
+F1E3,,futbol football soccer ball
```

Comment:
As far as I know "futbol" is not used in English, but per #5 I thought an additive change would be better than replacing it.